test/governancestaking/GovernanceStaking.test.ts

```diff
@@ -305,5 +305,16 @@ describe("GovernanceStaking - staking with GD  and get Rewards in GDAO", () => {
     await governanceStaking.withdrawStake("100")
   })
 
+  it("it should be withdraw without issue when staking amount is too large",async()=>{
```

Comment:
rename test, what it should be checking is
1. accumlated per share has enough precision when reward << totalproductivity
2. user receive fractional gdao when his stake << totalProductivity